<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="3" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(80, 80, 80);
  border-radius: 0
px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QFrame" name="frameGrip">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25,25,25);
  border-radius: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="headerFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignRight|Qt::AlignTop">
        <widget class="QFrame" name="btnframe">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QPushButton" name="bt_min">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border: 1px solid transparent;

}
QPushButton:hover {
  background-color: #148CD2;    
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/icon1/minus.svg</normaloff>:/icon1/minus.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="bt_close">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border: 1px solid transparent;

}
QPushButton:hover {
  background-color: #148CD2;    
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/icon1/x.svg</normaloff>:/icon1/x.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabGen">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>25</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Default Client :</string>
           </property>
          </widget>
          <widget class="QComboBox" name="cbCList">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>101</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pbApply">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>170</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pbCancel">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>170</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>30</y>
             <width>161</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Retail_Last_ClientCode</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>171</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Allow Only INDEX TO Trade</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>110</y>
             <width>231</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Overide OrderType Rule For IndexOption </string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStts">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <widget class="QCheckBox" name="cxbTrdSts">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Trade_socket</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cxbOrdSts">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Order_socket</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbSttsApply">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>230</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbSttsCancel">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>230</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cxbPOSts">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Place_Order</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <widget class="QCheckBox" name="cxbOrdLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Order_socket</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cxbTrdLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Trade_socket</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbSttsApply_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cxbPosLog">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Position_socket</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbSttsCancel_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabBeep">
       <attribute name="title">
        <string>beep</string>
       </attribute>
       <widget class="QCheckBox" name="cxbOrdBeep">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>on_Order</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbBeepApply_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cxbTrdBeep">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>on_Trade</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbBeepCancel_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cxbPOBeep">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>on_place_order</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
